src/nebula-hooks/use-context-menu.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore  TODO: remove tag when onContextMenu is declared in nebula/stardust
```

Comment:
Are you agree with this ? We will remove it soon after fixing the sn-table version

Reply:
It looks fine. What do you mean by fixing the sn-table version? I think it will be like this until onContext is add `onContextMenu` tot the nebula api